README.md

```diff
@@ -91,122 +91,195 @@ It is recommended to build the code in a separate directory from the source dire
-	int domain_extent_y(P) ;
-  data:
-	domain_x = 0, 16 ;
-	domain_y = 0, 0 ;
-	domain_extent_x = 16, 14 ;
-	domain_extent_y = 30, 30 ;
-  } // group bounding_boxes
+This should produce the following output (stdout) as well as two files, `partition_mask_2.nc` and `partition_metadata_2.nc`.
 
-group: connectivity {
-  variables:
-	int top_neighbours(P) ;
-	int top_neighbour_ids(T) ;
-	int top_neighbour_halos(T) ;
-	int bottom_neighbours(P) ;
-	int bottom_neighbour_ids(B) ;
-	int bottom_neighbour_halos(B) ;
-	int left_neighbours(P) ;
-	int left_neighbour_ids(L) ;
-	int left_neighbour_halos(L) ;
-	int right_neighbours(P) ;
-	int right_neighbour_ids(R) ;
-	int right_neighbour_halos(R) ;
-  data:
-	top_neighbours = 0, 1 ;
-	top_neighbour_ids = 0 ;
-	top_neighbour_halos = 30 ;
-	bottom_neighbours = 1, 0 ;
-	bottom_neighbour_ids = 1 ;
-	bottom_neighbour_halos = 30 ;
-	left_neighbours = 0, 0 ;
-	right_neighbours = 0, 0 ;
-  } // group connectivity
-}
+```
+$ mpirun -n 2 ../decomp -g test_2.nc -x m -y n -o 'yx' -m "land_mask"
+Partitioning total time: 9.2725e-05 (secs)
+Partitioning Statistics:
+ Total weight of dots = 12
+ Weight on each proc: ave = 6, max = 6, min = 6
+ Maximum weight of single dot = 1
+ Send count: ave = 0, min = 0, max = 0
+ Recv count: ave = 0, min = 0, max = 0
+ Max dots: ave = 6, min = 6, max = 6
+ Max memory: ave = 8, min = 8, max = 8
+ # of OverAlloc: ave = 0, min = 0, max = 0
+ Median find iteration counts:
+   Serial iterations per process: avg 0.000000 min 0 max 0
+   Parallel iterations:
+     Per process: avg 1.000000 min 1 max 1
+     Total for all parallel cuts: 1
+     Detail of first cuts:
+       Level 1 cut count: avg 1.000000 variance 0.000000 actual: 1
+ Start-up time (secs): ave = 2.70065e-05, min = 2.6947e-05, max = 2.7066e-05
+ Start-up time (%): ave = 29.1254, min = 29.0612, max = 29.1895
+ Pre-median time (secs): ave = 4.311e-06, min = 4.034e-06, max = 4.588e-06
+ Pre-median time (%): ave = 4.64923, min = 4.3505, max = 4.94796
+ Median time (secs): ave = 8.2655e-06, min = 8.066e-06, max = 8.465e-06
+ Median time (%): ave = 8.91399, min = 8.69884, max = 9.12915
+ Comm time (secs): ave = 5.0859e-05, min = 5.0418e-05, max = 5.13e-05
+ Comm time (%): ave = 54.8493, min = 54.3737, max = 55.3249
+ STATS Runs 1  bal  CURRENT 1.000000  MAX 1.000000  MIN 1.000000  AVG 1.000000
+ STATS Runs 1  moveCnt CURRENT 0.000000  MAX 0.000000  MIN 0.000000  AVG 0.000000
+ STATS DETAIL count:  min 6  max 6  avg 6.000000  imbal 1.000000
+ STATS DETAIL wdim = 0; no detail available
+```
+
+The `decomp` tool produces two netCDF-4 files (using the classic data model) named `partition_mask_<num_mpi_processes>.nc` and `partition_metadata_<num_mpi_processes>.nc` with the following layout:
 
+- `partition_mask_<num_mpi_processes>.nc` - can be used to check the partitioning
+- `partition_metadata_<num_mpi_processes>.nc` - is used by neXtSIM_DG to read in the domain decomposition information, such as
+  domain sizes, neighbours, halos etc.
```

Comment:
You say "the following layout" but then don't actually mention layout. I think you could just drop everything from "named" to "the following layout" anyway.

Reply:
thanks done in [cea56b4](https://github.com/nextsimhub/domain_decomp/pull/58/commits/cea56b4d24313d498c7648ddbd8b6e489cee0311)